program/src/balance.rs

```diff
@@ -219,40 +228,44 @@ mod test {
             (0, Lamports(0))
         );
     }
+
     #[test]
-    fn get_target_balance_weighted_validators() {
-        let mut validators = Validators::new_fill_default(2);
-        validators.entries[0].entry.weight = Weight(1000);
-        validators.entries[1].entry.weight = Weight(3000);
+    fn get_target_balance_inactive_validator() {
+        // 200 Lamports delegated, but only one active validator,
+        // so all of the target should be with that one validator.
+        let mut validators = Validators::new_fill_default(3);
+        validators.entries[0].entry.stake_accounts_balance = Lamports(100);
+        validators.entries[1].entry.stake_accounts_balance = Lamports(0);
```

Comment:
If we set this to a non-zero value, that would have caught the bug above where inactive validators are not considered for the current balance.